src/Config.php

```diff
@@ -380,4 +380,65 @@ public function setAlternativeCachePath(string $path, array $applyForCaches = nu
         }
         return $this;
     }
+
+    /**
+     * Set PHP configuration settings.
+     *
+     * Example:
+     * \B13\Config::get()->setPhpSettings([
+     *      'max_execution_time' => 1000,
+     *      'max_input_time' => 1000,
+     *      'post_max_size' => '100M',
+     *      'upload_max_filesize' => '100M',
+     *  ]);
+     *
+     * @param array $settings An associative array of PHP settings.
+     * @return $this
+     */
+    public function setPhpSettings(array $settings): self
+    {
+        foreach ($settings as $key => $value) {
+            ini_set($key, $value);
+        }
+
+        return $this;
+    }
+
+    /**
+     * Override or append TYPO3 configuration settings for a given path with key-value pairs.
+     *
+     * Examples:
+     * \B13\Config::get()->overrideSetting(['SYS'], ['mySetting' => 'new-value']);
+     * \B13\Config::get()->overrideSetting(
+     *     ['EXTENSIONS', 'your_extension_key'],
+     *     [
+     *         'allowedIp' => 'xxx.xxx.xxx.xxx',
+     *         'contactPid' => '404',
+     *         'mode' => 'foreign',
+     *         'notificationRecipients' => 'mymail@domain.com',
+     *         'remoteHost' => 'xxx.eu',
+     *     ]
+     * );
+     *
+     * @param array $configPath An array representing the path to the configuration option.
+     * @param array $keyValuePairs An associative array of key-value pairs to set within the specified path.
+     * @return $this
+     */
+    public function setConfigPathValues(array $configPath, array $keyValuePairs): self
```

Comment:
instead of the configPath as array, we could use "SYS" or "EXTENSIONS/your_extension_key" so it is the same syntax as the ExtensionConfiguration class from TYPO3.

Also, instead of dealing with the array manipulation we could simply use a `array_replace_recursive($TYPO3_CONF_VARS['SYS'], $keyValuePairs)`

What do you think?

Reply:
I like that. Stay tuned 😄